daml-lf/transaction-scalacheck/src/main/scala/com/digitalasset/daml/lf/value/ValueGenerators.scala

```diff
@@ -196,6 +196,14 @@ object ValueGenerators {
     } yield ValueTextMap[ContractId](SortedLookupList(Map(list: _*)))
   def valueMapGen: Gen[ValueTextMap[ContractId]] = valueMapGen(0)
 
+  private def valueGenMapGen(nesting: Int) =
+    for {
+      list <- Gen.listOf(for {
+        k <- Gen.lzy(valueGen(nesting)); v <- Gen.lzy(valueGen(nesting))
```

Comment:
```suggestion
        Gen.zip(Gen.lzy(valueGen(nesting)), Gen.lzy(valueGen(nesting)))
```

Without the `for` trappings.